README.md

```diff
@@ -14,6 +14,21 @@ A repository for hosting nextflow [`DSL2`](https://www.nextflow.io/docs/edge/dsl
     * [Uploading to `nf-core/modules`](#uploading-to-nf-coremodules)
 * [Help](#help)
 
+## Terminology
+
+The DSLv2 features for Nextflow are new for everyone and not just beginners. We discussed some terminology terms to discuss things related to modules & subworkflows more appropriately:
+
+* *Module*: An "atomic" module (one thing), e.g. a reusable process from an old pipeline that can be used between different pipelines
```

Comment:
* *Module*: A `process` that can be used in different workflows, which is atomic (i.e. can/should not be divided further).